qiskit_ibm_runtime/options/zne_options.py

```diff
@@ -124,10 +124,11 @@ class ZneOptions:
             * ``"fallback"``, which simply returns the raw data corresponding to the lowest noise
               factor (typically ``1``) without performing any sort of extrapolation.
 
-        If more than one extrapolator is specified, the ``evs`` and ``stds`` reported in the
-        result's data refer to the first one, while the extrapolated values
-        (``evs_extrapolated`` and ``stds_extrapolated``) are sorted according to the order of
-        the extrapolators provided.
+        The extrapolated values (``evs_extrapolated`` and ``stds_extrapolated``) are sorted according to
+        the order of the provided extrapolators. If more than one extrapolator is specified, the ``evs``
+        and ``stds`` reported in the result's data refer to the first successful extrapolator, where an
+        extrapolator success is determined heuristically, in particular the standard deviation must be
+        smaller than ``0.25``.
```

Comment:
I'd consider not locking into particular details of heuristics unless the entire thing is explained in detail.

```suggestion
        extrapolator success is determined heuristically.
```